data_portal_api.yml

```diff
@@ -198,17 +332,28 @@ paths:
         A schema that contains a field dictating the visibility of the deployments that will either make the project
         visible on the public sites or only viewable through obfuscated URLs.
       parameters:
-        - $ref: '#/components/parameters/submission_uuid'
+        - $ref: '#/components/parameters/project_uuid'
       requestBody:
         content:
           application/json:
             schema:
               description: >-
-                a schema that has fields for all available project-level metadata.
+                a schema that contains a field dictating the visibility of the deployments that will either make the
```

Comment:
I think you might have interpreted this too literally - the text was explaining what the parameters here should be which is literally a single enum dictating whether to publish the project privately or publicly.

Reply:
yeah, a bit too litteral on my part.